### PR TITLE
Feature/demo home

### PR DESCRIPTION
**Pull request by zhangjing28**
## Description

* SBT demo home page and upload page
* [Design Link](https://www.figma.com/file/9nAYHsYlfHmyT8RL5P8ISZ/Manta-Pay?node-id=2231%3A384&t=OVv14iUt6cTatSAY-0)

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [X] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [X] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

File: src/resources/images/sbt/index.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import img1 from './1.png';
```
Comment (zhangjing28) [inline src/resources/images/sbt/index.ts:1]: These png files will be removed when our thrid party's API is ready.

File: src/pages/SBTPage/components/UploadPanel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  const { imgList, setImgList } = useSBT();
+  const onImageChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e?.target?.files?.length) {
+      setImgList([...imgList, ...e.target.files]);
+    }
+  };
+  return (
+    <div className="relative w-max">
+      <div className="border border-dashed bg-primary rounded-lg w-48 h-48 flex justify-center items-center">
+        <svg
```
Comment (zhangjing28) [inline src/pages/SBTPage/components/UploadPanel/index.tsx:15]: This `svg` will move to a single file after [icon refactor PR](https://github.com/Manta-Network/manta-front-end/pull/355) is merged.

File: src/pages/SBTPage/components/UploadPanel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      </p>
+      <div className="grid w-full gap-6 grid-cols-5 pb-24 mt-6">
+        {imgList?.map((img, index) => {
+          return (
+            <div className="relative w-max group" key={index}>
+              <img
+                src={URL.createObjectURL(img)}
+                className="rounded-lg w-48 h-48"
+              />
+              <svg
```
Comment (zhangjing28) [inline src/pages/SBTPage/components/UploadPanel/index.tsx:69]: Same as above.

File: src/pages/SBTPage/SBTContext/index.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <SBTContext.Provider value={value}>{props.children}</SBTContext.Provider>
+  );
+};
+
+export const useSBT = () => {
+  const data = useContext(SBTContext);
+  if (!data || !Object.keys(data)?.length) {
+    throw new Error(
+      'useSBT() can only be used inside of <Web3Context />, please declare it at a higher level.'
```
Comment (Kevingislason) [inline src/pages/SBTPage/SBTContext/index.tsx:46]: This comment doesn't make sense to me, I haven't ever seen `Web3Context`. Is this something you are going to add later?

**Review by Kevingislason — APPROVED** [review]

File: src/pages/SBTPage/SBTContext/index.tsx (hunk 4 above)
Comment (zhangjing28) [reply src/pages/SBTPage/SBTContext/index.tsx:46]: My bad, This should be `<SBTContext />`

Comment (zhangjing28) [reply src/pages/SBTPage/SBTContext/index.tsx]: fixed

**Review by Kevingislason — APPROVED** [review]

**Pull request merged**